src/PollingBlockTracker.ts

```diff
@@ -6,12 +6,13 @@ import { BaseBlockTracker, Provider } from './BaseBlockTracker';
 const createRandomId = getCreateRandomId();
 const sec = 1000;
 
-interface PollingBlockTrackerArgs {
-  provider: Provider;
-  pollingInterval: number;
-  retryTimeout: number;
-  keepEventLoopActive: boolean;
-  setSkipCacheFlag: boolean;
+export interface PollingBlockTrackerOptions {
+  provider?: Provider;
```

Comment:
Should this be left as mandatory? Since an error is thrown if it's missing anyway